unit-testing/presentation.html

```diff
@@ -1,27 +1,398 @@
+          <li class="fragment">Testing statistic</li>
+          <li class="fragment">Testing libraries</li>
+          <li class="fragment">Jest as a testing framework</li>
+          <li class="fragment">Husky</li>
+        </ul>
+      </section>
+      <!-- 2 slide -->
+      <section>
+        <h3 class="slide-name">Testing role in Software Development​</h3>
+        <ul>
+          <li class="fragment">Testing is about verifying that what was specified is what was delivered.</li>
+          <li class="fragment">The testing is important since it discovers defects/bugs before the delivery to the
+            client, which guarantees the quality of the software.​</li>
+        </ul>
+      </section>
+      <!-- 3 slide -->
+      <section>
+        <h3 class="slide-name">Major benefits of software testing​</h3>
+        <ul>
+          <li class="fragment">Launch a flawless software application</li>
+          <li class="fragment">Prevent targeted malware attacks</li>
+          <li class="fragment">Improves the user experience</li>
+          <li class="fragment">Curtail maintain costs</li>
+          <li class="fragment">Improve time-to-market</li>
+          <li class="fragment">Enhance quality of apllication</li>
+          <li class="fragment">Boost competitiveness</li>
+        </ul>
+      </section>
+      <!-- 4 slide -->
+      <section>
+        <h3 class="slide-name">The scope of Testing​</h3>
+        <ul>
+          <li class="fragment"> The test scope defines what areas, features, and functionalities of an application
+            should be tested</li>
+        </ul>
+      </section>
+      <!-- 5 slide -->
+      <section>
+        <h3 class="slide-name">Testing Pyramid​​</h3>
+        <img src='assets/images/testing_pyramid.svg' height="500" />
+      </section>
+      <!-- 6 slide -->
+      <section>
+        <h3 class="slide-name">Benefits of unit tests​​</h3>
+        <ul>
+          <li class="fragment">Save development time</li>
+          <li class="fragment">Provide quick access to code documentation</li>
+          <li class="fragment">Contribute to code quality​</li>
+          <li class="fragment">Improve application architecture​</li>
+          <li class="fragment">Debug process​</li>
+          <li class="fragment">Reduce costs in long run</li>
+        </ul>
+      </section>
+      <!-- 7 slide -->
+      <section>
+        <h3 class="slide-name">Impacts of neglecting unit tests</h3>
+        <ul>
+          <li class="fragment">The risks of simple mistakes leaking to the end user significantly increase in the
+            absence of unit tests.</li>
+          <li class="fragment">Skipping unit tests quickly turns fresh code into legacy code, which is harder to
+            maintain.</li>
+        </ul>
+      </section>
+      <!-- 8 slide -->
+      <section class="table-slide">
+        <h3 class="slide-name">TDD VS BDD</h3>
+        <table class="fragment" data-fragment-index="2">
```

Comment:
you have two tables here..
please find and fix all issues it should help to fix issue connected with safari

Reply:
done1, but didn't help to fix Safari